src/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
+  <section className="section">
+    <p className="box is-size-3">
+      Please get your
+      {' '}
```

Comment:
Use template string.

[More info.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Check other similar cases.

Reply:
We have no expressions in this component to use template literals. It's only a text. But I can use some tags to make it more readable.